src/dom/DomEvent.Pointer.js

```diff
@@ -1,15 +1,9 @@
-import * as DomEvent from './DomEvent';
-import Browser from '../core/Browser';
 import {falseFn} from '../core/Util';
 
-/*
- * Extends L.DomEvent to provide touch support for Internet Explorer and Windows-based devices.
- */
-
-var POINTER_DOWN =   Browser.msPointer ? 'MSPointerDown'   : 'pointerdown';
-var POINTER_MOVE =   Browser.msPointer ? 'MSPointerMove'   : 'pointermove';
-var POINTER_UP =     Browser.msPointer ? 'MSPointerUp'     : 'pointerup';
-var POINTER_CANCEL = Browser.msPointer ? 'MSPointerCancel' : 'pointercancel';
+var POINTER_DOWN =   'pointerdown';
```

Comment:
There's no point on leaving the uppercase "constants" defined like this. The values should be refactored into the code IMO: replace all occurences of `POINTER_DOWN` with `'pointerdown`, etc.

Reply:
Done :+1: